spec/features/user_sees_own_profile_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+feature 'user sees profile' do
+  let(:user) { FactoryGirl.create(:user) }
+  scenario 'user clicks name on topbar' do
+    visit root_path
+    sign_in_as(user)
+    save_and_open_page
+    click_link "Signed in as #{user.username}"
+    expect(page).to have_content(user.email)
+
+    end
```

Comment:
`end` at 12, 4 is not aligned with `scenario 'user clicks name on topbar' do` at 5, 2.